### ISSUE TITLE
When is synth-shell-prompt.root.config used?

### ISSUE THREAD

**Issue by ga2k** [andresgongora/synth-shell#222]
Hi, great work.

When is synth-shell-prompt.root.config used in place of synth-shell-prompt.config ?

I saw the red prompt once, but in my re-jigging to get things arranged to be synched across multiple computers, only the non-root one is ever used. No amount of logging in as root, 'sudo su'ing or whatever will now activate it.

Cheers,
Geoff.

**Comment by ga2k** [comment]
Yes, .bashrc, .bash_aliases .bash_logout and .profile are all symlinked  from the onedrive folder to /root/ and to each user's home folder, the bin folder where the script lives is symlinked, and the .config folder is too.

**Comment by andresgongora** [comment]
I meant, does root's .bashrc contain the source-hooks for synth-shell?

Being symlinked is no problem in my experience :)

**Comment by ga2k** [comment]
Yes it does.

**Comment by ga2k** [comment]
I've removed everything and re-installed it for "all users" and that works so I'm leaving sleeping dogs lie :)

**Comment by andresgongora** [comment]
Perfect! Glad you got it working :)
